cartographer_ros/cartographer_ros/node.cc

```diff
@@ -113,6 +114,9 @@ Node::Node(
   constraint_list_publisher_ =
       node_handle_.advertise<::visualization_msgs::MarkerArray>(
           kConstraintListTopic, kLatestOnlyPublisherQueueSize);
+  tracked_pose_in_map_publisher_ =
+      node_handle_.advertise<::geometry_msgs::PoseStamped>(
+          kTrackedPoseTopic, kLatestOnlyPublisherQueueSize);
```

Comment:
Don't advertise if the feature is disabled by the node options.

Reply:
done